Source/Dafny/Verifier/Translator.ExpressionTranslator.cs

```diff
@@ -18,6 +18,18 @@ internal class ExpressionTranslator {
         // below), but also, the debugger may invoke HeapExpr and that will cause an increment as well.
         get { Statistics_HeapUses++; return _the_heap_expr; }
       }
+
+      /// <summary>
+      /// Return HeapExpr as an IdentifierExpr.
+      /// CAUTION: This getter should be used only if the caller "knows" that HeapExpr really is an IdentifierExpr.
+      /// </summary>
+      public Boogie.IdentifierExpr HeapCastToIdentifierExpr {
```

Comment:
Honestly this I would just inline since the comments do not provide more information than `(Boogie.IdentifierExpr)HeapExpr` already would.